astropy/io/ascii/core.py

```diff
@@ -277,6 +277,9 @@ class BaseInputter:
     encoding = None
     """Encoding used to read the file"""
 
+    cache = True
```

Comment:
This is the magic to make the `ascii.cds` example cache the files. Since `encoding` is implemented as a class attribute, it is only natural `cache` is one too. But doing it this way, I don't know how to let user change it to `True` only when they want to and have the default to `False`. While setting to `True` here works, it is an API change.